src-nextgen/loaders/baremetal/x86_32/grub-legacy/debug/dbgapi.c

```diff
@@ -1,172 +1,189 @@
-/*-

```

Comment:
src-nextgen/loaders/baremetal/x86_32/grub-legacy/debug/dbgapi.c has to be consistent with 
uxmhf/uobjcoll/uobjs/main/uobjs/debug/dbgapi.c in https://github.com/uberspark/uberxmhf/pull/26
Can you please sync?